src/N3Parser.js

```diff
@@ -835,25 +867,21 @@ export default class N3Parser {
     return this._readPath;
   }
 
-  // ### `_readRDFStarTailOrGraph` reads the graph of a nested RDF* quad or the end of a nested RDF* triple
-  _readRDFStarTailOrGraph(token) {
-    if (token.type !== '>>') {
-      // An entity means this is a quad (only allowed if not already inside a graph)
-      if (this._supportsQuads && this._graph === null && (this._graph = this._readEntity(token)) !== undefined)
-        return this._readRDFStarTail;
-      return this._error(`Expected >> to follow "${this._object.id}"`, token);
-    }
-    return this._readRDFStarTail(token);
-  }
-
   // ### `_readRDFStarTail` reads the end of a nested RDF* triple
```

Comment:
Do you really mean the deprecated `RDF*`, or should this name the current `RDF-star`?
```suggestion
  // ### `_readRDFStarTail` reads the end of a nested RDF-star triple
```